internal/pkg/agent/cmd/install.go

```diff
@@ -227,6 +233,14 @@ func installCmd(streams *cli.IOStreams, cmd *cobra.Command) error {
 	return nil
 }
 
+func validateBasePath(basePath string) error {
+	if !filepath.IsAbs(basePath) {
+		return fmt.Errorf("base path %q is not absolute", basePath)
+	}
+
+	return nil
+}
```

Comment:
[Suggestion]
I don't think a function just to call another function is necessary, using directly `filepath.IsAbs` is even more explicit about what is being validated.